src/mixins/filing-template-mixin.ts

```diff
@@ -304,7 +303,7 @@ export default class FilingTemplateMixin extends DateMixin {
         legalType: this.getEntitySnapshot.businessInfo.legalType
       },
       restoration: {
-        approvalType: this.getStateFilingRestoration?.approvalType,
+        approvalType: this.getRestoration.approvalType,
```

Comment:
This is a small fix, but unfortunately, this value is always "registrar" because the ApprovalType component does not emit the updated value -- that's a separate ticket.